interview-questions.md

```diff
@@ -7,44 +7,78 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: cohort model runs through students id and db migrate the new foreign key into the data
 
-  Researched answer:
+  Researched answer: a change can be made by running the rails generate migration. adding a column add column students, cohort id ,integer. then running db migrate
```

Comment:
And the foreign key will always live on the belongs_to model.

Reply:
I'll add this in so i don't forget thank you.